readthedocs/projects/tasks.py

```diff
@@ -621,6 +621,11 @@ def finish_build(version_pk, build_pk, hostname=None, html=False,
         version.built = True
         version.save()
 
+    if not pdf:
```

Comment:
The main purpose of this PR is to fix the problem that there might be PDF versions lying around for `latest` which are not up to date because the PDF builds started failing or the project admin disabled PDF builds. That is a very annoying side effect. We shall never leave out-dated versions of docs around. That is super confusing.

We can address clearing the PDFs for all versions in a different issue if you think it's worth it.